synapse/lib/storm.py

```diff
@@ -153,3 +153,24 @@ class SudoCmd(Cmd):
     def runStormCmd(self, snap, genr):
         snap.elevated = True
         yield from genr
+
+class CountCmd(Cmd):
+    '''
+    Iterate through query results, and print the resulting number of nodes
+    which were lifted. This does yield the nodes counted.
+
+    Example:
+
+        foo:bar:size=20 | count
+
+    '''
+    name = 'count'
+
+    def runStormCmd(self, snap, genr):
+
+        i = None
+        for i, node in enumerate(genr, 1):
+            yield node
+
+        if i:
```

Comment:
If we default i to zero, we can always print a statement which could be `Counted 0 nodes`.

Reply:
Yea, lets default i to 0